Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -518,6 +596,17 @@ namespace O3DE::ProjectManager
         projectActionButton->setMenu(menu);
     }
 
+    void ProjectButton::ResetButtonWidgets()
+    {
+        HideContextualLabelButtonWidgets();
+        SetLaunchingEnabled(false);
+        SetProjectBuilding(false);
+
+        m_projectImageLabel->GetDownloadMessageLabel()->setVisible(false);
+        m_projectImageLabel->GetProgressPercentage()->setVisible(false);
+        m_projectImageLabel->GetProgressBar()->setVisible(false);
```

Comment:
if we don't set the progress percent to 0 here, will there be a momentary full or partial progress bar until a download progress event is received?